crates/amplifier-ingester/Cargo.toml

```diff
@@ -13,7 +13,33 @@ supervisor.workspace = true
 tracing.workspace = true
 eyre.workspace = true
 infrastructure = { workspace = true, features = ["consumer-interfaces"] }
+tokio.workspace = true
+tokio-util.workspace = true
+serde.workspace = true
+clap.workspace = true
+ctrlc.workspace = true
+toml.workspace = true
+url.workspace = true
+bin-util.workspace = true
+
+
 futures.workspace = true
 
+[features]
+default = ["gcp"]
```

Comment:
you have it as gcp here. But actually lets leave it as gcp and update Readme stating gcp is default one and nats is optional